infrastructure/main.tf

```diff
@@ -2,6 +2,12 @@ provider "azurerm" {
   features {}
 }
 
+provider "azurerm" {
+  features {}
+  alias           = "private_dns"
+  subscription_id = var.private_dns_subscription_id
```

Comment:
this needs to be set, it should always be the one for DTS-CFTPTL-INTSVC as private endpoints zones only have one set, possibly we should move the sandbox zones into the same place as it would make this easier